javatests/arcs/sdk/ReadSDKPerson.kt

```diff
@@ -0,0 +1,23 @@
+package arcs.sdk
+
+import kotlinx.coroutines.runBlocking
+
+class ReadSDKPerson : AbstractReadSDKPerson() {
```

Comment:
Not a huge deal, but Abbreviations/Acronyms should only have their first character capitalized. (e.g. `ReadSdkPerson`)